client/my-sites/plugins/index.js

```diff
@@ -76,6 +76,14 @@ module.exports = function() {
 			pluginsController.plugin
 		);
 
+		if ( config.isEnabled( 'automated-transfer' ) ) {
+			page( '/plugins/:plugin/:site_id/eligibility',
```

Comment:
We agreed to postpone this for future PRs, until we decide on appropriate path scheme.